### PR TITLE
commit yarn.lock changes

### PR DESCRIPTION
**Pull request by pro-wh**
we recently merged a big feature update with changes to the package.json. this PR applies the changes to yarn.lock that yarn makes when running `yarn`.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6646,294 +4070,17 @@ pkg-dir@^3.0.0:
...
-    source-map-js "^1.0.2"
-
-prebuild-install@^5.3.3:
-  version "5.3.4"
-  resolved "https://registry.yarnpkg.com/prebuild-install/-/prebuild-install-5.3.4.tgz#6982d10084269d364c1856550b7d090ea31fa293"
-  integrity sha512-AkKN+pf4fSEihjapLEEj8n85YIw/tN6BQqkhzbDc0RvEZGdkpJBGMUYx66AAMcPG2KzmPQS7Cm16an4HVBRRMA==
+prebuild-install@^5.3.3:
+  version "5.3.4"
+  resolved "https://registry.yarnpkg.com/prebuild-install/-/prebuild-install-5.3.4.tgz#6982d10084269d364c1856550b7d090ea31fa293"
+  integrity sha512-AkKN+pf4fSEihjapLEEj8n85YIw/tN6BQqkhzbDc0RvEZGdkpJBGMUYx66AAMcPG2KzmPQS7Cm16an4HVBRRMA==
```
Comment (pro-wh) [inline yarn.lock:4083]: why is this in the diff, these are identical

File: yarn.lock [hunk 2]
```diff
@@ -7317,61 +4326,26 @@ rxjs@6:
...
-    ajv-keywords "^5.0.0"
-
-select-hose@^2.0.0:
-  version "2.0.0"
-  resolved "https://registry.yarnpkg.com/select-hose/-/select-hose-2.0.0.tgz#625d8658f865af43ec962bfc376a37359a4994ca"
-  integrity sha1-Yl2GWPhlr0Psliv8N2o3NZpJlMo=
+safe-buffer@~5.1.0, safe-buffer@~5.1.1:
+  version "5.1.2"
+  resolved "https://registry.yarnpkg.com/safe-buffer/-/safe-buffer-5.1.2.tgz#991ec69d296e0313747d59bdfd2b745c35f8828d"
+  integrity sha512-Gd2UZBJDkXlY7GbJxfsE8/nvKkUEU1G38c1siN6QP6a9PT9MmHB8GnpscSmMJSoF8LOIrt8ud/wPtojys4G6+g==
```
Comment (pro-wh) [inline yarn.lock:4337]: yarn decided to reorder this

**Comment by pro-wh** [comment]
analysis of actual additions

- secp256k1 5.0.0
  - elliptic 6.5.4
    - bn.js 4.12.0
    - brorand 1.1.0
    - hash.js 1.1.7
    - hmac-drbg 1.0.1
    - minimalistic-crypto-utils 1.0.1
  - node-addon-api 5.1.0
  - node-gyp-build 4.6.0
- js-sha256 0.8.0

**Comment by lukaw3d** [comment]
Related to https://github.com/oasisprotocol/ledger-js/pull/627

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**